app/components/Project/About/About.css

```diff
@@ -15,9 +15,13 @@
   font-size: 0.9rem;
 }
 
-.propertyBlock,
+.propertyBlock {
+  padding-bottom: 25px;
+}
+
 .tagViewer {
   padding-bottom: 25px;
+  float: left;
```

Comment:
This is causing the "Edit Details" button to display on a separate line from the tags:
![image](https://github.com/StatTag/StatWrap/assets/967970/ce888159-eee1-47b9-84ef-fdd5ffd1a8e4)

We had wanted that all on one line to save whitespace, but open to discussion if you think there's a reason to keep them separated.
![image](https://github.com/StatTag/StatWrap/assets/967970/54b46d71-363b-4f30-84dd-5c1896139132)